frontend/src/metrics.js

```diff
@@ -1,4 +1,4 @@
-const testPilotGA = require('testpilot-ga');
+const testPilotGA = require('testpilot-ga/src/TestPilotGA');
```

Comment:
Totally, just wanted to point out that due to the fact of loose dependency versions (`"testpilot-ga": "^0.3.0"`), semver, and upstream repos, that could technically break at any point in the future when @chuckharmston publishes a new version of **testpilot-ga**.